### PR TITLE
Added case 2 questions to the server room

### LINKED ISSUE #105: Case 2: Add Questions to server room
## Expected Behaviour
Loading level two should drop sherry into the server room to answer questions. 
## Actual Behaviour
There is no level two
## Steps to Reproduce Behaviour

1. go to the main menu

### PR DESCRIPTION
**Pull request by Alice-Jaffray**
**Description of work.**

Added questions to the server room for case 2

**To test:**

Open the server room and check the question objects

Fixes #105

### CODE REVIEW

**Comment by cailafinn** [comment]
This branch needs to have it's conflicts resolved before review

**Comment by cailafinn** [comment]
I am getting this error when attempting to open the `ServerRoom` scene: 
`The file 'Assets/Rooms/ServerRoom.unity' seems to have merge conflicts. Please open it in a text editor and fix the merge.`

It may need rebasing instead of this merge commit?

**Review by cailafinn — CHANGES_REQUESTED** [review]
Questions look good and I like the new assets. Just a few notes:
- The canvases should be scaled using the "Anchor Presets" (click the square to the left of the `PosX` and `Width` boxes; Hold `alt` and `Shift` and click the bottom right object to stretch the canvas to the full area) and the Scale needs resetting to 1 for all values
- The buttons need assigning their `OnClick` methods. (Click the button, scroll down to the `OnClick()` section. Drag the object in the scene that is interacted with onto the Object section. Then function is then `ObjectInteraction > Correct()` for the right answer and `ObjectIneraction > Incorrect(Button)` for the wrong ones. The button you're assigning is then dragged onto the parameter

**Review by cailafinn — CHANGES_REQUESTED** [review]
Questions and new assets look good. Only note is that `ServerQuestion (2)` is too small and needs to be stretched like the rest. 

I also want to check if it should be possible to press 'E' to interact with the objects, as that is not currently working if the scene is played?

File: project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs [hunk 1]
```diff
@@ -41,7 +41,9 @@ public class MainMenuController : MonoBehaviour
     }
 
     public void level2() {
-        
+        ScoreManager.Instance.Reset();
```
Comment (cailafinn) [inline project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:48]: Needs to include a call to the level manager to set the current level (see above `level1()` method)

**Review by cailafinn — CHANGES_REQUESTED** [review]
Nearly perfect! Just a couple of little things:
- The level 2 main menu button needs to have `Interactable` ticked
- The `serverTower` in the top left seems to have been assigned the wrong question canvas

File: project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs (hunk 1 above)
Comment (Alice-Jaffray) [reply project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:48]: I don't see what you mean

Comment (Alice-Jaffray) [reply project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs:44]: I don't see what you mean

**Review by cailafinn — APPROVED** [review]
Looks good! Thanks for pushing this through :)

**Pull request merged**